apps/site-app/src/containers/TopUp/Purchase/eTransferModal.tsx

```diff
@@ -34,27 +33,28 @@ export const ETransferModal: FunctionComponent<Props> = (props: Props) =>
       isOpen={props.showDlg}
       header={haveTransferCodeHeader}
       okCallback={props.onCloseDlg}
-      progressPercent={1}
+      closeIcon={true}
+      closeIconFct={props.onCloseDlg}
     >
-      <Grid>
+      <Grid stackable columns={2}>
         <Grid.Row>
           <FormattedMessage {...yourTransferCode} />
         </Grid.Row>
-        <Grid.Row columns="two">
-          <Grid.Column className={styles.fixedColumn}>
```

Comment:
Those classes where nowhere in the style file

Reply:
converting to `less` has broken our styles type-checking.  We need to look into that.  Could you create an issue for that as well?

Reply:
The compiler should tell us if we reference non-existent styles.